src/components/Button.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+function Button({className,onclick}) {
+    return (
+        <div>
+            <button
+                className={className}
+                onClick={onclick}>
+            </button>
+
+        </div>
+    );
+}
```

Comment:
Dit is nu meer een doorgeefluik. Ten opzichte van het html button element zelf gebeurd er verder niets in dit component. Je had dus net zo goed direct kunnen gebruiken

Reply:
Wanneer je dit had gedaan had je je tekst nog gezien. Nu doe je namelijk niets met die prop
```suggestion
function Button({className,onclick, children}) {
    return (
        <div>
            <button
                className={className}
                onClick={onclick}>
                {children}
            </button>

        </div>
    );
}
```